frontends/web/src/routes/account/account.tsx

```diff
@@ -231,12 +225,12 @@ class Account extends Component<Props, State> {
         if (this.state.fatalError) {
             return;
         }
-        apiPost(`account/${this.props.code}/export`).then(exported => {
+        accountApi.exportAccount(this.props.code).then(exported => {
             this.setState({ exported });
-        });
+        }).catch(console.error);
```

Comment:
Would love to see these errors surfaced to the user, in the future.

Reply:
yes 👍